pkg/client_data/lib/package_api.dart

```diff
@@ -132,10 +132,14 @@ class VersionInfo {
   @JsonKey(name: 'archive_url')
   final String archiveUrl;
 
+  /// This is an optional field of the API response, it may be `null` or omitted.
+  final DateTime created;
```

Comment:
Should we consider using the word `published` instead?

@sigurdm, thoughts?

(I think API changes are worth bikeshedding a little, because they are super hard to reverse).

Reply:
I like `published`

Reply:
or "uploaded"...

Reply:
Okay, let's go with `published`.